tests/core/test_exp_manager.py

```diff
@@ -91,11 +91,11 @@ def __init__(self, *args, **kwargs):
 
     def train_dataloader(self):
         dataset = OnesDataset(2)
-        return torch.utils.data.DataLoader(dataset, batch_size=2)
+        return torch.utils.data.DataLoader(dataset, batch_size=2, num_workers=8)
 
     def val_dataloader(self):
         dataset = OnesDataset(10)
-        return torch.utils.data.DataLoader(dataset, batch_size=2)
+        return torch.utils.data.DataLoader(dataset, batch_size=2, num_workers=8)
```

Comment:
Are workers necessary for the test ?

Reply:
Not mandatory, but lots of unnecessary warnings with workers=0.